### ISSUE TITLE
Schema error with writing to `organizations` table

### ISSUE THREAD

**Issue by cayleyh** [singer-io/tap-pipedrive#19]
Just started seeing this today:

```
Error persisting data for table "organizations": Record 0 did not conform to schema:
#: 2 schema violations found
#: required key [timeline_last_activity_time] not found
#: required key [timeline_last_activity_time_by_owner] not found
```

Now the error has changed to:

```
2018-04-19 21:49:56,973Z   main - INFO Running tap-pipedrive version 0.0.11 and target-stitch version 1.7.0
2018-04-19 21:49:57,099Z   main - DEBUG Getting initial state
2018-04-19 21:49:57,157Z   main - INFO Starting tap: tap-env/bin/tap-pipedrive --config /tmp/tap_config.json --state /tmp/tap_state.json
2018-04-19 21:49:57,167Z   main - INFO Starting target: target-env/bin/target-stitch --config /tmp/target_config.json
2018-04-19 21:49:57,514Z    tap - Traceback (most recent call last):
2018-04-19 21:49:57,514Z    tap -   File "tap-env/bin/tap-pipedrive", line 7, in <module>
2018-04-19 21:49:57,515Z    tap -     from tap_pipedrive.cli import main
2018-04-19 21:49:57,515Z    tap -   File "/code/orchestrator/tap-env/lib/python3.5/site-packages/tap_pipedrive/cli.py", line 4, in <module>
2018-04-19 21:49:57,515Z    tap -     from tap_pipedrive.tap import PipedriveTap
2018-04-19 21:49:57,515Z    tap -   File "/code/orchestrator/tap-env/lib/python3.5/site-packages/tap_pipedrive/tap.py", line 10, in <module>
2018-04-19 21:49:57,515Z    tap -     from .streams import (CurrenciesStream, ActivityTypesStream, FiltersStream, StagesStream, PipelinesStream,
2018-04-19 21:49:57,515Z    tap - ImportError: No module named 'tap_pipedrive.streams'
2018-04-19 21:49:57,633Z target - INFO Using Stitch import URL https://api.stitchdata.com/v2/import/batch
2018-04-19 21:49:57,633Z target - INFO Exiting normally
2018-04-19 21:49:57,690Z   main - INFO Target exited normally with status 0
2018-04-19 21:49:57,691Z   main - INFO Exit status is: Tap failed with code 1. Target succeeded.
```

I can't reproduce the first error, but when I install the latest 0.0.11 tap and run locally, I can reproduce the above.

**Comment by cayleyh** [comment]
@dmosorast I just started seeing this @ about 1am this morning (PST). As of an hour ago I started getting the `ImportError` instead. I've been able to reproduce by updating my local copy from 0.0.10 -> 11 and running it locally against our live pipedrive account.

**Comment by cayleyh** [comment]
Can confirm that the .tar.gz that `pip` fetches does not include the `tap_pipdrive/streams` directory.

**Comment by dmosorast** [comment]
Hi @cayleyh, thanks for reporting this. The packaging issue should be resolved in 0.0.12.

As for the first issue, I see that those fields are marked as required in the [schema](https://github.com/singer-io/tap-pipedrive/blob/master/tap_pipedrive/schemas/recents/dynamic_typing/organizations.json#L200-L201). If you can run the tap locally successfully, it would be worth it to check the tap's output for the organizations stream to see if some records are erroneously missing those fields.

**Comment by suzybates** [comment]
Hello, I am not a developer, but I am using the Pipedrive stitch connection.  I am seeing this error as of 5 days ago.  is that anything I can do to resolve this issue for my integration without running code?
thank you so much for creating this connection!

**Comment by dmosorast** [comment]
@suzybates @cayleyh Since this tap was submitted by a community member, I'm not sure exactly why these fields are marked as required. However, they do not show up in the data from a test trial Pipedrive account either. I suspect they may have been either removed by Pipedrive, or a common customization that doesn't apply to all accounts.

I'm working up a quick fix in #21 to remove these fields from being required, which should unblock the `organizations` stream (and the `persons` stream after it, which also is missing these fields).
